index.php

```diff
@@ -1,3 +1,20 @@
+<?php
+
+require 'functions.php';
+
+$db = displayPDO();
+
+    $stmt = $db->query("SELECT `title` ,`intro_text` FROM `intro`");
+    $title_array = $stmt->fetch();
+    $stmt = $db->query("SELECT `paragraph_1`, `image`, `paragraph_2` FROM `about_me`");
+    $about_me_array = $stmt->fetch();
+    $stmt = $db->query("SELECT `title`, `image`, `paragraph` FROM `projects`");
+    $projects_array = $stmt->fetch();
```

Comment:
this only allows you to have 1 project shown on your portfolio, and it will always be the oldest project, not the newest one